website/manual.md

```diff
@@ -698,6 +698,86 @@ export main() {
 }
 ```
 
+### Executable scripts
+
+Deno provides ability to easily distribute executable code via `deno install`
+command.
+
+`deno install [EXE_NAME] [URL] [FLAGS...]` will install script available at
+`URL` with name `EXE_NAME`.
+
+This command is a thin wrapper that creates executable shell scripts which
+invoke `deno` with specified permissions and CLI flags.
+
+Example:
+
+```
+$ deno install file_server https://deno.land/std/http/file_server.ts --allow-net --allow-read
+> [1/1] Compiling https://deno.land/std/http/file_server.ts
+>
+> ✅ Successfully installed file_server.
+> /Users/deno/.deno/bin/file_server
+```
+
+By default scripts are installed at `$HOME/.deno/bin` and that directory must be
+added to the path manually.
+
+```
+$ echo 'export PATH="$HOME/.deno/bin:$PATH"' >> ~/.bashrc
+```
+
+Installation directory can be changed using `-d/--dir` flag:
+
+```
+deno install --dir /usr/local/bin prettier https://deno.land/std/prettier/main.ts --allow-write --allow-read
+```
+
+#### Flags and permissions
```

Comment:
I don't like all these subsubsections - it makes the ToC too complex. Leave this out and write it like a single essay.